compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -3463,6 +3465,28 @@ private STNode parseFuncCall(STNode identifier) {
         return STNodeFactory.createFunctionCallExpressionNode(identifier, openParen, args, closeParen);
     }
 
+    /**
+     * Parse error constructor expression.
+     *
```

Comment:
Shall we add the grammar rule here as a doc comment? You can check other expressions for a sample.

Reply:
Done